README.md

```diff
@@ -21,3 +21,5 @@ A normal first session might look like this:
 But, you can do so much more than that...your phone is not just a play thing anymore.
 
 This app is fully open source.  You can find our code and file issues [here](https://github.com/CypherpunkArmory/UserLAnd/).
+
+The assets that UserLAnd depends on and the scripts that build them are contained in other repositores.  For example, the common assets that are used for all distros and application are found at https://github.com/CypherpunkArmory/UserLAnd-Assets-Support .  For any specific distro or application, they are found at `https://github.com/CypherpunkArmory/UserLAnd-Assets-DISTRO_OR_APP_NAME` ( ie https://github.com/CypherpunkArmory/UserLAnd-Assets-Debian )
```

Comment:
I think this would look cleaner with some relevant inline links and more whitespace, such as

common assets ... are found in our [support repository](https://github.com/CypherpunkArmory/UserLAnd-Assets-Support)

Distribution specific assets can be found under CypherpunkArmory/UserLAnd-Assets-(__Distribution/App__). For example, our Debian specific assets can be found at [CypherpunkArmory/UserLAnd-Assets-Debian](https://github.com/CypherpunkArmory/UserLAnd-Assets-Debian)